llvm/test/CodeGen/Hexagon/iss127296.ll

```diff
@@ -0,0 +1,22 @@
+; RUN: llc -mtriple=hexagon -O0 < %s | FileCheck %s
+; REQUIRES: asserts
```

Comment:
Why does this need asserts?

Reply:
The original failure (#127296) happens with assertions enabled.  I removed it from my latest patch.  But I just tried a build without asserts and it looks like the codegen is no different with this fix.  So IIUC that means I want the asserts back in this test case, right?

Reply:
You want the test to always run, even if it only reproduces the original issue in some configuration. You should only use REQUIRES: asserts if you are checking for debug output only available in assertion-enabled builds.